fireapp/lib/presentation/volunteer_information/volunteer_information.dart

```diff
@@ -0,0 +1,204 @@
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Email: ${volunteerInformation.email}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Mobile No: ${volunteerInformation.mobileNo}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Preferred Hours: ${volunteerInformation.prefHours}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Experience Years: ${volunteerInformation.expYears}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text('Volunteer Possible Roles: ${volunteerInformation.possibleRoles.join(", ")}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Qualifications: ${volunteerInformation.qualifications.join(", ")}',
+                      style: TextStyle(
+                        fontSize: 14.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Availabilities:',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: [
+                        Text(
+                          'Monday: ${volunteerInformation.availabilities.Monday?.join(", ")}',
```

Comment:
This code will be directly printing out a list of numbers which will be completely meaningless to anyone using the app. There should be some kind of message if a person has no availability (rather than a blank list). You should look through the existing website codebase to figure out what the numbers mean and how to present them (here is a starting point -> https://github.com/TechlauncherFireApp/FireApp3.0/blob/06e0958b190ea44dc68fe998a687b7d9307ab404/ui/src/routes/Volunteers/Availability/Availability.js)